english/lighter/grab-lighter.md

```diff
@@ -1,10 +1,12 @@
-You swipe the lighter off the floor as your hand shakes. You try feverishly 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.